frontend/src/libs/helpers/test-server-error-type/test-server-error-type.helper.ts

```diff
@@ -0,0 +1,38 @@
+import { ServerErrorType } from '~/libs/enums/enums.js';
+import {
+  type ServerCommonErrorResponse,
+  type ServerValidationErrorResponse,
+} from '~/libs/types/types.js';
+
+import { objectPick } from '../helpers.js';
+
+// This is to reassure this key name at the typescript level
+const errorTypeKey: keyof (
+  | ServerCommonErrorResponse
+  | ServerValidationErrorResponse
+) = 'errorType';
```

Comment:
create separate type for `keyof (
  | ServerCommonErrorResponse
  | ServerValidationErrorResponse
)` and remove comment

Reply:
And why we have 2 different types?